integrated_channels/integrated_channel/models.py

```diff
@@ -65,13 +67,27 @@ class Meta:
         verbose_name_plural = _("Enterprise Customer Plugin Configurations")
         unique_together = (("enterprise_customer_uuid", "enterprise_integrated_channel"),)
 
+    @property
+    def enterprise_customer(self):
+        """
+        Return Enterprise Customer associated with this instance.
+
+        Return :class:`enterprise.models.EnterpriseCustomer` instance associated with this
+        :class:`EnterpriseCustomerPluginConfiguration` instance via uuid.
+        """
+        try:
+            return EnterpriseCustomer.objects.get(uuid=self.enterprise_customer_uuid)  # pylint: disable=no-member
+        except EnterpriseCustomer.DoesNotExist:
+            return None
```

Comment:
This seems like an odd scenario to me -- we have a UUID but the record does not exist in the system -- I would consider logging this situation instead of simply sweeping it under the rug